### DISCUSSION TITLE
YOLOv4

### DISCUSSION THREAD

**Discussion by janhvis51** [community/community#123232] [Programming Help]
### Select Topic Area

Question

### Body

 I'm
encountering a challenging error in my YOLOv4 project, and I was
hoping you might be able to offer some guidance or assistance.

I've been working on implementing YOLOv4 for object detection in a
plant health detection in real time project, and while I've made
progress, I've hit a roadblock with a particular error.

Code LIne:!./darknet detector train data/obj.data
cfg/yolov4-custom.cfg /mydrive/yolov4/backup/yolov4-obj_last.weights
-dont_show -map

Error statement: Allocate additional workspace_size = 319.68 MB
Loading weights from
/mydrive/yolov4/backup/yolov4-obj_last.weights...Couldn't open file:
/mydrive/yolov4/backup

It seems that there is no yolov4-obj_last.weights in the backup
folder. the path of backup provided in the obj.data file is correct. 
@glenn-jocher 
@apoon2

**Comment by MostlyKIGuess** [comment]
Can you send the structure of your project? 
Better if you just send the code, it will be easier to understand.

**Reply by janhvis51** [reply]
This is our code
https://colab.research.google.com/drive/1G76q_y8UyBuieyTqZMf6BEOvsOxcvcal#scrollTo=bqnhwroGehnt
this the article we are following
https://medium.com/analytics-vidhya/train-a-custom-yolov4-object-detector-using-google-colab-61a659d4868

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @janhvis51 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information. 

Happy coding @janhvis51 and @MostlyKIGuess !!
